app/services/api/v1x0/catalog/copy_portfolio_item.rb

```diff
@@ -18,11 +18,11 @@ def initialize(params)
 
         def process
           PortfolioItem.transaction do
-          @new_portfolio_item = make_copy
-        rescue StandardError => e
-          Rails.logger.error("Failed to copy Portfolio Item #{@portfolio_item.id}: #{e.message}")
-          raise
-        end
+            @new_portfolio_item = make_copy
```

Comment:
No need to make `@new_portfolio_item` instance variable. Local variable is sufficient.

Reply:
No, it's needed by https://github.com/RedHatInsights/catalog-api/blob/master/app/controllers/api/v1x0/portfolio_items_controller.rb#L47